koku/masu/database/sql/openshift/reporting_ocp_volume_summary_by_project_p.sql

```diff
@@ -49,11 +51,14 @@ INSERT INTO {{schema | sqlsafe}}.reporting_ocp_volume_summary_by_project_p (
         {{source_uuid}}::uuid as source_uuid,
         max(cost_category_id) as cost_category_id,
         max(raw_currency) as raw_currency,
-        sum(distributed_cost) as distributed_cost
+        sum(distributed_cost) as distributed_cost,
+        persistentvolumeclaim,
+        storageclass
     FROM {{schema | sqlsafe}}.reporting_ocpusagelineitem_daily_summary
     WHERE usage_start >= {{start_date}}::date
         AND usage_start <= {{end_date}}::date
         AND source_uuid = {{source_uuid}}
         AND data_source = 'Storage'
-    GROUP BY usage_start, cluster_id, cluster_alias, namespace, cost_model_rate_type
+        AND persistentvolumeclaim IS NOT NULL -- coming from platform distributed
```

Comment:
```suggestion
        AND persistentvolumeclaim IS NOT NULL
```
Can you explain what this comment means?

Reply:
Yeah, so this is actually fall out from the fix I added for this PR:
- https://github.com/project-koku/koku/pull/4404

Problem number 2 to be exact. I added the data_source so that we could distribute the volume cost to the user projects. However, as a result we get a null for the `persistentvolumeclaim` for the platform distributed which was causing some wonkiness for the capacity calculations. So, I just removed it storage calculations.